debug/org.eclipse.debug.tests/src/org/eclipse/debug/tests/viewer/model/VisibleVirtualItemValidator.java

```diff
@@ -46,11 +46,12 @@ public int getLength() {
 	@Override
 	public boolean isItemVisible(VirtualItem item) {
 		int position = 0;
```

Comment:
```suggestion
		int depth= 0;
```

Reply:
It's actually about some kind of position and not about depth, as the index within a parent is considered as well.